src/router/index.js

```diff
@@ -16,7 +16,6 @@ import Issues from '../components/Issues/Issues.vue';
 import Velocity from '../components/Sprints/Velocity.vue';
 import Burndown from '../components/Sprints/Burndown.vue';
 import IssuesGraphic from '../components/Issues/IssuesGraphic.vue';
-import Grade from '../components/Projects/Grade.vue';
```

Comment:
This entire module can be removed, since GPA has been deprecated in CC.

Reply:
Right? @MatheusRich

Reply:
Yeah. Maybe later we add something else.

Reply:
Should I also remove this file then?